grails-app/views/campaigns/showModules/_campaignDataDatePublished.gsp

```diff
@@ -13,9 +13,11 @@
                     <g:render template="/campaigns/showModules/campingModalCreateSummoning" model="[]"/>
                 </g:if>
 
-                <g:link mapping="politicianCampaignStatsShow" params="[campaignId: campaign.id]" role="button"
-                        class="edit"><span class="fal fa-chart-line fa-2x"></span><span
-                        class="sr-only">Stats</span></g:link>
+                <g:if test="${campaign.statsEnabled}">
```

Comment:
This logic will be independent of editable flag

Reply:
Moved the editable flag to make it independent.